src/utils/index.js

```diff
@@ -101,6 +102,13 @@ const getErrorMessages = (error) => {
   return ['Unknown error.'];
 };
 
+const isNonExemptChanged = (initialRuns, currentRuns, runKey) => {
```

Comment:
I always feel weird about method names with negations, but this is fine, just is a bit hard to read when you go through something like ```if not isNonExemptChanged```